src/DataGrid.tsx

```diff
@@ -149,6 +152,7 @@ export interface DataGridProps<R, SR = unknown, K extends Key = Key> extends Sha
   selectedRows?: Maybe<ReadonlySet<K>>;
   /** Function called whenever row selection is changed */
   onSelectedRowsChange?: Maybe<(selectedRows: Set<NoInfer<K>>) => void>;
+  isRowSelectable?: Maybe<(row: NoInfer<R>) => boolean>;
```

Comment:
I wonder if this should be named `isRowSelectionDisabled/Enabled` instead.
For example, a row can be selected, but toggling it may be disabled/read-only.

Reply:
Maybe we rename it to `isRowSelectionDisabled`